.cirrus.yml

```diff
@@ -989,6 +989,61 @@ test_image_build_task:
     env:
         DRYRUN: 1
 
+fcos_image_build_task: &fcos_image-build
+    name: "Build FCOS image $CTXDIR"
+    alias: fcos_image_build
+    # Some of these container images take > 1h to build, limit
+    # this task to a specific Cirrus-Cron entry with this name.
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: $CIRRUS_CRON == ''
+    timeout_in: 120m  # emulation is sssllllooooowwww
+    gce_instance:
+        <<: *standardvm
+        image_name: build-push-${IMAGE_SUFFIX}
+        # More muscle required for parallel multi-arch build
+        type: "n2-standard-4"
+    matrix:
+        - env:
+            CTXDIR: contrib/podman-next/fcos-podmanimage
+    env:
+        DISTRO_NV: "${FEDORA_NAME}"  # Required for repo cache extraction
+        PODMAN_USERNAME: ENCRYPTED[b9f0f2550029dd2196e086d9dd6c2d1fec7e328630b15990d9bb610f9fcccb5baab8b64a8c3e72b0c1d0f5917cf65aa1]
+        PODMAN_PASSWORD: ENCRYPTED[e3444f6072853f0c8db7f964ead5e2204116af485469fa0de367f26b9316b460fd842a9882f552b9e9a83bbaf650d8b4]
+        CONTAINERS_USERNAME: ENCRYPTED[54a372d5f22f424173c114c6fb25c3214956cad323d5b285c7393a71041884ce96471d0ff733774e5dab9fa5a3c8795c]
+        CONTAINERS_PASSWORD: ENCRYPTED[4ecc3fb534935095a99fb1f2e320ac6bc87f3e7e186746e41cbcc4b5f5379a014b9fc8cc90e1f3d5abdbaf31580a4ab9]
+    main_script:
+        - set -a; source /etc/automation_environment; set +a
+        - podman build -t quay.io/podman/fcos:podman-next $CTXDIR
+
+
+test_fcos_image_build_task:
+    <<: *fcos_image-build
+    alias: test_fcos_image_build
+    # Allow this to run inside a PR w/ [CI:BUILD] only.
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: $CIRRUS_PR != '' && $CIRRUS_CHANGE_TITLE =~ '.*CI:BUILD.*'
+    # This takes a LONG time, only run when requested.  N/B: Any task
+    # made to depend on this one will block FOREVER unless triggered.
+    # DO NOT ADD THIS TASK AS DEPENDENCY FOR `success_task`.
+    trigger_type: manual
+    # Overwrite all 'env', don't push anything, just do the build.
+    env:
+        DRYRUN: 1
```

Comment:
```suggestion
```

These aren't needed...unless you want to use them in `runner.sh` (might be a good idea actually, easy way for the script to tell which "mode" it's operating in).

Reply:
Sorry.  I was intending "mode" to mean (for example) `$DRYRUN==0` (or undefined) vs `$DRYRUN==1`.  The build-push scripting is sensitive to it, but you're scripts aren't (currently).  If it's not needed it would be best to drop it.  It's presence may confuse maintainers in the future.

Reply:
ack thanks. I removed the test task entirely. I can add it later if need be.